sdk/managedservices/azure-mgmt-managedservices/tests/test_mgmt_managedservices.py

```diff
@@ -19,50 +19,60 @@ def setUp(self):
 
     def test_managedservices_crud(self):
         scope = 'subscriptions/38bd4bef-41ff-45b5-b3af-d03e55a4ca15'
-        assignmentid = '367257d7-b69a-4b76-9c18-6dd3e9ae33ce'
-        api_version = '2018-06-01-preview'
-        definitionid = '1cf1704a-f8a8-4082-a8be-457d3f68e812'
+        assignmentid = 'c0d5f994-63a1-484d-8c1e-a2ac825efd60'
+        api_version = '2019-06-01'
+        definitionid = '8daec8c7-7567-47ff-9009-f0a4ec429a3c'
         regdef_string='{"registrationDefinitionName":"Registration Test","description":"dpp","managedByTenantId":"bab3375b-6197-4a15-a44b-16c41faa91d7","authorizations":[{"principalId":"d6f6c88a-5b7a-455e-ba40-ce146d4d3671","roleDefinitionId":"acdd72a7-3385-48ef-bd42-f606fba81ae7"}]}'
         properties = json.loads((regdef_string))
+        sleep=15
 
         #create definition
-        defresponse = self.client.registration_definitions.create_or_update(
-                    definitionid, api_version, scope, properties)
-        definitionresourceid = defresponse.id
-        self.assertIsNotNone(defresponse)
-        self.assertEqual(definitionid, defresponse.name)
+        response = self.client.registration_definitions.create_or_update(definitionid, api_version, scope, properties)
```

Comment:
api_version should not be a parameter of the operation, but a constant in the main client. This is likely a Swagger issue that wasn't caught by the Swagger reviewer.

Reply:
merge from https://github.com/Azure/azure-sdk-for-python/tree/restapi_auto_managedservices/resource-manager and you should not have to define api_version ever again